airflow/cli/commands/connection_command.py

```diff
@@ -73,13 +85,22 @@ def connections_get(args):
 @suppress_logs_and_warning
 def connections_list(args):
     """Lists all connections at the command line"""
+    airflow_console = AirflowConsole()
+    if CUSTOM_BACKEND:
+        airflow_console.print(
+            (
+                "WARNING: The Airflow CLI will not return Connections or "
+                "Variables stored in an alternative secrets backend."
+            ),
+            style="magenta",
+        )
```

Comment:
At that moment airflow grab credentials/variables from SomeAlternateSecretsBackend, EnvironmentVariablesBackend and MetastoreBackend.

And actually it return list only from MetastoreBackend, and ignore all other backends.

Reply:
My fault, actually for listing connection and variable Airflow doesn't use `MetastoreBackend` rather than it make direct query to Metastore

Reply:
Yeah -- `connections_list` and `variables_list` doesn't capture secrets backend conns/vars:

```python
@suppress_logs_and_warning
def connections_list(args):
    """Lists all connections at the command line"""
    with create_session() as session:
        query = session.query(Connection)
        if args.conn_id:
            query = query.filter(Connection.conn_id == args.conn_id)
        conns = query.all()

        AirflowConsole().print_as(
            data=conns,
            output=args.output,
            mapper=_connection_mapper,
        )
```